package.json

```diff
@@ -12,7 +12,7 @@
     "copy:files": "copyfiles -u 1 ./src/config.json ./build/ && npm run copy:doxy",
     "copy:doxy": "copyfiles -u 2 src/doxy/* build/doxy/",
     "test": "npm run test:mocked && npm run test:server",
-    "test:server": "jest --silent --config=jest.server.config.js",
```

Comment:
We should probably keep the silent flag so it doesn't flood the CI build log with unnecessary information.

Reply:
fixed